docs/BOOTING.md

```diff
@@ -79,9 +79,45 @@ Second stage GRUB is expected to do all the [heavy lifting](../pkg/grub/rootfs.c
 an EVE instance and because it resides in IMGA or IMGB partitions it can easily be upgraded and patched.
 Behavior of this stage of boot process is controlled by a read-only grub.cfg under {IMGA,IMGB}/EFI/BOOT/
 but it can further be tweaked by the grub.cfg overrides on the CONFIG partition. Note that an override
-grub.cfg is expected to be a [complete override](../pkg/grub/rootfs.cfg#L132) and anyone constructing
+grub.cfg is expected to be a [complete override](../pkg/grub/rootfs.cfg#L143) and anyone constructing
 its content is expected to be familiar with the overall flow of read-only grub.cfg.
 
+Option to use in grub.cfg on the CONFIG partition:
+
+1. Cgroup related options. Please follow the [example](README.md#eve-cgroups) to write
+   these options in the correct variables:
+    1. `dom0_mem` - memory limit for eve cgroup (default is `750M,max:750M`)
+    2. `dom0_max_vcpus` - cpu limit for eve cgroup (default is `1`)
+    3. `eve_mem` - memory limit for cgroups with services of EVE (default is `600M,max:600M`)
+    4. `eve_max_vcpus` - cpu limit for cgroups with services of EVE (default is `1`)
+    5. `ctrd_mem` - memory limit for cgroups with containerd-shims of EVE (default is `350M,max:350M`)
+    6. `ctrd_max_vcpus` - cpu limit for cgroups with containerd-shims of EVE (default is `1`)
+2. Installer options. These options are used during [the installation](DEPLOYMENT.md) process and may be set by adding
+   line `set_global dom0_extra_args "$dom0_extra_args OPTION1=VAL1 OPTION2 "`, where OPTION fields described below
+   (`DISK` below is e.g. `sda`):
+    1. `eve_blackbox` - if set, installer will collect [information](../pkg/mkimage-raw-efi/install#L93)
+       from EVE to INVENTORY partition on installation media (it must be writable, i.e. USB flash drive)
+       and exit without installation.
+    2. `eve_nuke_disks=DISK` - clean partition table on the defined disks before install.
+       You can use multiple disks separated by comma. Please use this option if you know that other
```

Comment:
You should mention that this option is a one-shot. IOW, it can NOT be combined with any other installer option -- since none of the other installer actions are taken.

Reply:
This option do not lead to reboot/poweroff